modules/developer-cli-odo-ref-storage.adoc

```diff
@@ -0,0 +1,124 @@
+:_content-type: REFERENCE
+[id="odo-storage_{context}"]
+= odo storage
+
+
+`odo` lets users manage storage volumes that are attached to the components. A storage volume can be either an ephemeral volume using an `emptyDir` Kubernetes volume, or a link:https://kubernetes.io/docs/concepts/storage/volumes/#persistentvolumeclaim[Persistent Volume Claim] (PVC). A PVC allows users to claim a persistent volume (such as a GCE PersistentDisk or an iSCSI volume) without understanding the details of the particular cloud environment. The persistent storage volume can be used to persist data across restarts and rebuilds of the component.
```

Comment:
Refer to the comment above related to cross-references in the modules and do the needful.

Reply:
Again, this is an external link.